slices/util-linux.yaml

```diff
@@ -0,0 +1,130 @@
+package: util-linux
+
+essential:
+  - util-linux_copyright
+
+slices:
+  bins:
```

Comment:
What do you actually need from here? Since this is not a dependency for any of the above SDFs, I'm assuming you just need a subset of utilities.

If you add them all under `bins` (which is ok since this is a catch-all slice), then you need to test them all. If you only need a few, then please follow a similar approach to `coreutils`.

Reply:
I only need `su` from this package, so I'll create a separate individual slice for `su` there.